clients/src/main/resources/common/message/JoinGroupRequest.json

```diff
@@ -54,6 +56,9 @@
         "about": "The protocol name." },
       { "name": "Metadata", "type": "bytes", "versions": "0+",
         "about": "The protocol metadata." }
-    ]}
+    ]},
+    { "name": "Reason", "type": "string",
+      "versions": "8+", "nullableVersions": "8+","default": "null",
```

Comment:
nit: Could we put `versions` on the previous line to stay inline with the other fields? A space is missing before `default`.

Reply:
Could we also update `RequestResponseTest` to cover this change?